the_destination_of_shadow/DirectX.cpp

```diff
@@ -78,6 +113,54 @@ HRESULT DirectX::InitD3Device(HWND hWnd, CONST TCHAR* FilePath)
 	return S_OK;
 }
 
+KEY_STATE DirectX::GetInputState(BYTE curr_diks, BYTE prev_diks)
+{
+	if (IsKeyPressed(curr_diks)==TRUE)
```

Comment:
```suggestion
	if (IsKeyPressed(curr_diks))
```
とするのではなく、if文の中は明示的にTRUEと比較する方針ですか？
であれば、それで揃えて下さいね。

Reply:
後日中村さんと話し合いどちらにするかを決めます

Reply:
ちなみに、昔のC言語は言語としてboolが無かったので「TRUEが0と定義されているかも」という可能性がありました。(成功やエラー無しのときに `return 0;` と書くことが多い名残)
そのせいで「TRUE/FALSEと比較しましょう」なんていうルールを見たことありますが、今のCやC++では冗長なので、提示したコードの方が良いと思いますよ。

また、C/C++ではif文の条件式は非boolでも可能なので `if(curr_dik & 0x80)` なんてコードが書けるわけですが(JavaやC#ではコンパイル通りません)、これを「C/C++の便利機能なのでOK」とするか「明示的に `if((curr_dik & 0x80) != 0x00)` と書くべき」とするかは議論の余地がありますね。